test/utils/time-block-helper.js

```diff
@@ -0,0 +1,54 @@
+advanceTimeAndBlock = async (time) => {
```

Comment:
Is it the same as our ```scripts/utils/Timer.js```?

Reply:
These libraries are similar so I will include this in the same PR with the new formulas